src/database/migrations/20240405132731-add-notification-indexes.js

```diff
@@ -0,0 +1,44 @@
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    queryInterface.sequelize.transaction((t) =>
+      Promise.all([
+        queryInterface.addIndex(
+          "notifications", // name of Source model
+          ["site_id", "user_id"],
+          {
+            transaction: t,
+          }
+        ),
+        queryInterface.addIndex(
+          "notifications", // name of Source model
+          ["user_id"],
+          {
+            transaction: t,
+          }
+        ),
+      ])
```

Comment:
⚠️ Warning: this might not work. Creating indexes on table locks, and so typically you do not want to do it in parallel, and not in a transaction either.

I recommend you drop the transaction entirely and await each lock creation.

Same comment for the the removal